02Lesson/02Project/index.html

```diff
@@ -1,10 +1,66 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title></title>
+  <title>Daily Planet</title>
   <link rel="stylesheet" href="css/style.css"/>
 </head>
-<body>
-  <!-- You code start here -->
+<body background="https://static1.squarespace.com/static/550890c6e4b076f89f2184cf/t/5589d2b5e4b0872e7abdef3e/1435095762334/Austin+Cityscape+and+Lady+Bird+Lake+2.jpg?format=1500w">
+  <header>
+    <h1 style="text-align: center;">Daily Planet</h1>
+    <nav style="background-color: pink; text-align: center;">
+      <a href="http://www.espn.com/">Sports</a>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
+      <a href="http://www.statesman.com/austin-mugshots/#40">Local</a>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
+      <a href="http://economictimes.indiatimes.com/">Economy</a>
+    </nav>
+  </header>
+  <main>
+    <p style="font-family: DauphinPlain; font-size: 1.1em; color: blue; text-align: center;">Since October, 2016, Where you get your news when you don't want to read sad stuff!</p>
+    <section>
+      <div style="background-color: white;">
+        <p style="font-family: Oswald, Arial, sans-serif; letter-spacing: .2em;">TODAY'S TOP ARTICLE:</p>
+        <article>
+          <a href="http://www.vanityfair.com/culture/2001/03/furries200103">
+            <h2 style="letter-spacing: .25em; color: #089E00; font-family: EngraversGothicBT-Regular; text-transform: uppercase; font-weight: normal; text-align: center;">Furries To Take Over<br>Convention Center</h2>
+          </a>
+          <center><img class="resize1" src="http://www.gannett-cdn.com/-mm-/6e64d324fe70f636d63836629cecafcb26dd0365/c=58-0-4202-3116&r=x404&c=534x401/local/-/media/USATODAY/USATODAY/2014/12/07/635535832539370412-AP-APTOPIX-Hotel-Evacuation-Chlorine-Leak.jpg" alt="Furries getting jolly"></center>

```

Comment:
`center` element is obsolete in HTML5. Use CSS property `text-align` instead. Check errors in W3C validator in Atom. 
Child elements should also be indented.